modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/snapshot/SnapshotService.java

```diff
@@ -39,10 +39,7 @@ public void addSnapshotable(Snapshotable snapshotable) {
 
     public byte[] snapshot() {
         HashMap<String, Object[]> snapshots = new HashMap<String, Object[]>(snapshotableList.size());
-
-        if (log.isDebugEnabled()) {
-            log.debug("Taking snapshot ...");
-        }
+        log.info("Taking snapshot ...");
```

Comment:
I too think it should be debug. Usually if there's an issue and you need to debug it, you enable debug logs. Otherwise, it'll be noise.